pkg/util/crdutil.go

```diff
@@ -218,6 +218,25 @@ var (
 		Type:     "date",
 		JSONPath: ".metadata.creationTimestamp",
 	}
+
+	tidbMonitorAdditionalPrinterColumns      []extensionsobj.CustomResourceColumnDefinition
+	readyTiDBMonitorAdditionalPrinterColumns = extensionsobj.CustomResourceColumnDefinition{
+		Name:     "READY",
+		Type:     "string",
+		JSONPath: ".status.ready",
```

Comment:
```suggestion
		JSONPath: `.status.deployment.conditions[?(@.type=="Available")].status`,
```

can we think the TidbMonitor is ready when the underlying deployment's `Available` condition is true?